README.md

```diff
@@ -1,57 +1,53 @@
 # calculator-angular-tdd
 
-Construyendo una aplicación usando la metodología tdd o test driven development, intentando utilizar las mejores prácticas conocidas por los autores [JuanSeBestia](github.com/JuanSeBestia), [Jess casas](https://github.com/jlcasasg) 
+Builiding a app using the metology tdd or test driven development, We have used the best practice that thea autors know ([JuanSeBestia](github.com/JuanSeBestia), [Jess casas](https://github.com/jlcasasg))
 
+We espect have all mecanisme related of continuos integration and continus delivey CI/CD
 
-
-Se espera tener todos los mecanismos de pruebas, integración y despliegue con CI/CD
-
-* La documentación que no requiera gráfico se realizará en .md dentro de este git
+* The domcuntation that not require grahps would made wit MarckDown inside for this git. 
```

Comment:
the documentation that doesn't require graphs would be made with ... inside this git

Reply:
Done